src/classes/GRER_SyncGradeLevel_TDTM.cls

```diff
@@ -58,22 +61,53 @@ global with sharing class GRER_SyncGradeLevel_TDTM extends hed.TDTM_Runnable {
         //Declare and initiate lists to update
         List<Contact> contactsToUpdate = new List<Contact>();
         List<Grade_Enrollment__c> gradeEnrolsToUpdate = new List<Grade_Enrollment__c>();
+        
+        //Declare and initiate oldMap
```

Comment:
I know i add the comments here. I feel like we can remove 'Declare and initiate'. Sort of verbose...